CHANGELOG.md

```diff
@@ -89,6 +89,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 #### Added
 - [#143](https://github.com/shopsys/shopsys/pull/143) [Shopsys Coding Standards dev-master](./packages/coding-standards/) is now used
     - config [easy-coding-standard.yml](./easy-coding-standard.yml]) for importing rules of new easy-coding-standard package
+- [#249 first architectonical codesniff](https://github.com/shopsys/shopsys/pull/249)
```

Comment:
I do not want to be a nitpick but all the other entries (or most of them) in the changelog have the unified format - hash and number of the PR followed with dash and name of the PR starting with capital letter...Or, it seems like you can use just the hash and number (as you can see on the lines above) and github automatically displays the PR name in the output. But in that case, you should probably rename the PR (start with capital letter). Also, when I am mentioning the PR itself, please always try to add some description/reason of the PR in one sentence at least...